plugins/chanbackup.c

```diff
@@ -436,17 +436,30 @@ static struct command_result *after_listpeers(struct command *cmd,
 	bool is_connected;
         u8 *serialise_scb;
 
+	if (!peer_backup)
+		return notification_handled(cmd);
```

Comment:
Is there a reason to short circuit here rather than in `after_staticbackup`?

Reply:
I was aiming for minimal changes. but I pulled this out of the loop.  I think you're right, the check could be done earlier..